### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review this PR.

Changes:
- Store or save form data to localStorage
- Populate the form with data from local storage
- Handle form input changes and update formData
- Handle form reset and clear local storage
- Attach event listeners to input fields and buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,21 +211,25 @@ <h3>Terminal</h3>
...
-                        required>Enter text here</textarea>
+                    <textarea name="message" rows="4" cols="39" maxlength="500" id="message">Enter text here</textarea>
+                </div>
+                <div class="btns-div">
+                    <button type="submit" value="Submit">
+                        <h6>Get in touch</h6>
+                    </button>
+                    <button type="reset" id="resetButton">
+                        <h6>Reset</h6>
+                    </button>
```
Comment (V-Blaze) [inline index.html:230]: -  You have done great so far :+1: :+1: Form data are stored in local storage. :+1: :100: There are some changes you need to make before moving to the next milestone. 
- [ ] You did great adding the `reset button` and its functionality, but as required by Microverse and also in the project requirement, you are meant to stick to the Figma design as much as possible without adding or removing any design. I'll suggest you remove the Button as it's not part of the design. :+1: :100: :+1:

File: index.html [hunk 2]
```diff
@@ -211,21 +211,25 @@ <h3>Terminal</h3>
...
                 <div class="email-div">
-                    <input type="email" name="email" placeholder="Email address" id="email" required>
+                    <input type="email" name="email" placeholder="Email address" id="email">
                     <hr>
                 </div>
                 <div class="message-div">
-                    <textarea name="message" rows="4" cols="39" maxlength="500" id="message"
-                        required>Enter text here</textarea>
+                    <textarea name="message" rows="4" cols="39" maxlength="500" id="message">Enter text here</textarea>
+                </div>
```
Comment (V-Blaze) [inline index.html:223]: - [ ] In this message field, the `Enter text here` should be in a placeholder attribute inside the `textarea` tag and not as a text. I'll suggest you move it into a placeholder as it tends to show up as a value in the UI. :+1: :+1:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @egichuhis  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Meaningful commit messages✔
-  Clean and well documented code👋 
-  Correct Github flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rica213 — APPROVED** [review]
# 🟢 STATUS: APPROVED
Hi Edwin & Blessman 
My name is Larissa. I am your code reviewer 🥰

Your project is complete! You have completed all required changes from the previous reviewer. It's time to merge it 🛸 🛸 
Congratulations! 🎉 



## Highlights

- [ ] No Linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Descriptive commit messages ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**